Dynamo WONT CONVERT for torch.onnx.is_in_onnx_export

Found in detectron, small repro:
```
import torch
import torchdynamo

def my_compiler(gm, inputs):
    return gm.forward

@torchdynamo.optimize(my_compiler)
def fn(x, y):
    if torch.onnx.is_in_onnx_export():
        return x
    else:
        return y

fn(1, 2)
```
prints
```
torchdynamo.convert_frame: [ERROR] WON'T CONVERT fn /private/home/suo/scratch/test.py line 8
due to:
Traceback (most recent call last):
  File "/raid/suo/torchdynamo/torchdynamo/variables/tensor.py", line 258, in create
    assert (
AssertionError: torch.* op returned non-Tensor bool call_function <function is_in_onnx_export at 0x7f9032bcbc10>

from user code:
   File "/private/home/suo/scratch/test.py", line 10, in fn
    if torch.onnx.is_in_onnx_export():

Set torchdynamo.config.verbose=True for more information
==========
```

--- a/test/test_functions.py
+++ b/test/test_functions.py
@@ -153,6 +153,20 @@ class FunctionTests(torchdynamo.testing.TestCase):
         return sub(*args)
 
     @make_test
+    def test_is_in_onnx_export(x, y):
+        if torch.onnx.is_in_onnx_export():
+            return x - 1
+        else:
+            return y + 1
+
+    @make_test
+    def test_is_fx_tracing(x, y):
+        if torch.fx._symbolic_trace.is_fx_tracing():
+            return x - 1
+        else:
+            return y + 1
+
+    @make_test
     def test_listarg1(a, b):
         return torch.cat([a, b])
 

--- a/torchdynamo/config.py
+++ b/torchdynamo/config.py
@@ -46,6 +46,8 @@ class AccessLimitingConfig(ModuleType):
         torch.jit.is_scripting: False,
         torch.jit.is_tracing: False,
         torch._C._get_tracing_state: None,
+        torch.fx._symbolic_trace.is_fx_tracing: False,
+        torch.onnx.is_in_onnx_export: False,
     }
 
     # root folder of the project

